### PR TITLE
subsystem-bench: Remove benchmark_name from BenchmarkUsage

### PR DESCRIPTION
**Pull request by olaitanade**
resolves https://github.com/paritytech/polkadot-sdk/issues/3601
polkadot address: 5Gsej8PYq3fSrc6FvArzg1Wu3hGrKkStxTUAKDQrWeVVF8WP

### CODE REVIEW

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-clippy
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/5471512

**Comment by olaitanade** [comment]
bot fmt

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-clippy
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/5472482

**Comment by AndreiEres** [comment]
@olaitanade Thank you for the PR!
Please make CI happy and then I can review it.

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-clippy
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/5484934

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-clippy
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/5485080

**Comment by olaitanade** [comment]
@AndreiEres Thanks. Clippy is happy now. ready for review.

**Comment by AndreiEres** [comment]
> @AndreiEres Thanks. Clippy is happy now. ready for review.

Cool, looks like you formatted the code with different settings. Could you remove it?

**Comment by AndreiEres** [comment]
bot fmt

**Comment by olaitanade** [comment]
@AndreiEres, ready for review.

File: polkadot/node/subsystem-bench/src/cli/subsystem-bench.rs [hunk 1]
```diff
@@ -129,7 +129,6 @@ impl BenchCli {
 		for (index, CliTestConfiguration { objective, mut test_config }) in
 			test_sequence.into_iter().enumerate()
 		{
-			let benchmark_name = format!("{} #{} {}", &self.path, index + 1, objective);
```
Comment (AndreiEres) [inline polkadot/node/subsystem-bench/src/cli/subsystem-bench.rs:132]: We still need the benchmark name. Despite removing it from struct, the final output should be the same as before. Please add it to a `println!` that prints the usage. The same goes for printing in regression tests.

**Review by AndreiEres — CHANGES_REQUESTED** [review]
Looks good! Just a little change requested. Please, resolve conflicts with master

File: polkadot/node/subsystem-bench/src/cli/subsystem-bench.rs (hunk 1 above)
Comment (olaitanade) [reply polkadot/node/subsystem-bench/src/cli/subsystem-bench.rs]: @AndreiEres Thanks for the review. Please review my changes as requested. Thanks

**Comment by AndreiEres** [comment]
@olaitanade Please look at changes. There are lots of them that should be there.

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-clippy
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/5615555

**Pull request closed without merging**